src/shell.js

```diff
@@ -69,6 +69,8 @@ if (!ENVIRONMENT_IS_PTHREAD) ENVIRONMENT_IS_PTHREAD = false; // ENVIRONMENT_IS_P
 var PthreadWorkerInit; // Collects together variables that are needed at initialization time for the web workers that host pthreads.
 if (!ENVIRONMENT_IS_PTHREAD) PthreadWorkerInit = {};
 var currentScriptUrl = (typeof document !== 'undefined' && document.currentScript) ? document.currentScript.src : undefined;
+#else
+var ENVIRONMENT_IS_PTHREAD = false;
```

Comment:
Let's not add this in at all, in singlethreaded builds we want to never see a variable `ENVIRONMENT_IS_PTHREAD` anyhwere. I like to keep this strict to enforce a "don't pay for what you don't use" barrier on this feature.